code/datums/movement_modifier/movement_modifiers.dm

```diff
@@ -79,6 +80,7 @@
 // robot modifiers
 /datum/movement_modifier/robot_base
 	health_deficiency_adjustment = -INFINITY
+	mob_pull_multiplier = 0.2
```

Comment:
Add a comment to explain this deviation, ideally.

Reply:
Done